pkg/virt-launcher/virtwrap/nichotplug.go

```diff
@@ -113,8 +113,8 @@ func (vim *virtIOInterfaceManager) hotUnplugVirtioInterface(vmi *v1.VirtualMachi
 }
 
 func interfacesToHotUnplug(vmiSpecInterfaces []v1.Interface, domainSpecInterfaces []api.Interface) []api.Interface {
-	ifaces2remove := netvmispec.FilterInterfacesSpec(vmiSpecInterfaces, func(i v1.Interface) bool {
-		return i.State == v1.InterfaceStateAbsent
+	ifaces2remove := netvmispec.FilterInterfacesSpec(vmiSpecInterfaces, func(iface v1.Interface) bool {
+		return iface.State == v1.InterfaceStateAbsent && iface.SRIOV == nil
```

Comment:
I think you need to cover this condition in the unit tests (under `DescribeTable("domain interfaces to hot-unplug",`).

Reply:
Done